docs/BinSkimRules.md

```diff
@@ -870,3 +870,21 @@ This rule emits CSV data to the console for every compiler/language/version comb
 
 ---
 
+## Rule `BA6001.EnableSourceLink`
+
+### Description
+
+SourceLink information should be present in the PDB. This applies to binaries built with the C# and MSVC compilers. See https://aka.ms/sourcelink for more information.
+
+### Messages
+
+#### `Pass`: Pass
+
+The PDB for '{0}' contains SourceLink information.
+
+#### `Error`: Error
+
+The PDB for '{0}' does not contain SourceLink information.
```

Comment:
>The PDB for '{0}' does not contain SourceLink information [](http://example.com/codeflow?start=0&length=57)

Let's build out this description a tiny bit, see below for a start. What support, exactly, should we document for C++? Does SourceLink have a seamless interop story with existing source server tech, or does it replace it? Does it provide value because we're updating symbol servers to be source link aware? Just trying to clarify my understanding further in support of building a concise desc that answers some typical questions.

'SourceLink information should be present in the PDB of all binaries built with the C# and MSVC compilers. For .NET assemblies. Source Link executes when creating a NuGet package and embeds source control metadata inside assemblies and the package. Developers who download the package and have Source Link enabled in Visual Studio can step into its source code. Frictionless source-driven debugging provides a good user experience for dependency consumers and also accelerates security response in the  event of supply-chain compromise. See [https://aka.ms/sourcelink](https://aka.ms/sourcelink) for more information.'

Reply:
Done